test cases/common/262 configure file output format/compare.py

```diff
@@ -0,0 +1,5 @@
+import sys
+
+with open(sys.argv[1], 'r', encoding='utf-8') as f, open(sys.argv[2], 'r', encoding='utf-8') as g:
+    if f.read() != g.read():
+        sys.exit('contents are not equal')
```

Comment:
This file can be written shorter:
```
import sys
import filecmp

assert filecmp.cmp(sys.argv[1], sys.argv[2], shallow=False)
```

Reply:
Unfortunately, this doesn't work because of line endings on Windows.